TestModels/SimpleTypes/SimpleString/src/SimpleStringImpl.dfy

```diff
@@ -30,10 +30,10 @@ module SimpleStringImpl refines AbstractSimpleTypesSmithyStringOperations  {
         var res := GetStringOutput(value := input.value);
         return Success(res);
     }
-    method GetStringUTF8 ( config: InternalConfig,  input: GetStringInput )
-    returns (output: Result<GetStringOutput, Error>) {
+    method GetStringUTF8 ( config: InternalConfig,  input: GetStringUTF8Input )
```

Comment:
There should be a `GetStringUTF8KnownValue` endpoint as well.

This catches some very subtle possible bugs in the to and from that cancel out. We have this in most of the simple types. Given how Go handles strings, I think this would be a good addtion.